chart/values.yaml

```diff
@@ -0,0 +1,169 @@
+# Default values for kube-httpcache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: quay.io/mittwald/kube-httpcache
+  pullPolicy: IfNotPresent
+  tag: "stable"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Enable StatefulSet or Deployment
+statefulset:
+  enabled: false
+
+deployment:
+  enabled: true
+
+# Enable configMap for Varnish Template File
+# Otherwise use extravolume if the file is too big
+configmap:
+  enabled: true
+
+# kube-httpcache specific configuration
+cache:
+  # name of frontend service
+  frontendservice: kube-httpcache
+  # name of backend service
+  backendservice: backend-service
+  # Varnish storage backend type (https://varnish-cache.org/docs/trunk/users-guide/storage-backends.html)
+  varnishstorage: malloc # default,malloc,umem,file...
+  # Varnish storage backend size
+  storagesize: 128M # K(ibibytes), M(ebibytes), G(ibibytes), T(ebibytes) ... unlimited
+  # ConfigMap name for Varnish template
+  configmapname: vcl-template
+  # Secret Name for Varnish admin credentials
+  secretname: varnish-secret
+  # File name for Varnish template
+  template: varnish.tpl
```

Comment:
Many of these properties will create resources of the same name and will prevent this Chart from being installed multiple times in the same namespace. I think it might be sensible to fall back on the `fullname` template for many of these.

Reply:
the K8S resource names are now removed as values.